### PR TITLE
Update basics.md

### PR DESCRIPTION
**Pull request by snakecase**
Add an example of Windows environmental variable to the "Customizing the editor" section.

### CODE REVIEW

**Comment by snakecase** [comment]
I hope this saves a visit to Google for those like me who are not sure if this env variable works on Windows too.

**Comment by smeech** [comment]
Makes sense. Thank you.

File: docs/configuration/basics.md [hunk 1]
```diff
@@ -102,12 +102,18 @@ the default system editor (Notepad on Windows and Nano on Unix systems) will be
...
+environment variable to the path of your desired editor.
 
+Linux/macOS example:
 ```
 EDITOR=/usr/bin/vim
 ```
 
+Windows example:
+
+![SystemPropertiesAdvanced_2024-05-16_02-08-15](https://github.com/espanso/website/assets/4579891/09e0c67e-a22e-4da0-a764-c9077db32aa9)
```
Comment (AucaCoyan) [inline docs/configuration/basics.md:114]: Hi! Can I bother you asking where this link comes from? I don't see it on the `espanso/website` repo, maybe you missed adding this file?

Comment (snakecase) [reply docs/configuration/basics.md:114]: Hm I thought GitHub would handle this automatically so I just pasted the image into text editing form. Still opens on my side but I guess I have to put the image in the repo separately. Brb.

Comment (snakecase) [reply docs/configuration/basics.md:114]: Nvm I will cancel this PR and do this properly.

Comment (AucaCoyan) [reply docs/configuration/basics.md:114]: There was no need! If you copy and paste the image inside of `/static/img/something.png` it was fine for me!

Comment (snakecase) [reply docs/configuration/basics.md:114]: GitHub's Web interface is PITA for such things so I just cloned the repo and did this locally. See https://github.com/espanso/website/pull/63.

**Pull request closed without merging**